### PR TITLE
Scamark ingredients

### PR DESCRIPTION
**Pull request by aleene**
**Checklist:**
Added missing ingredients from the Scamark import

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -14555,7 +14555,7 @@ carbon_footprint_fr_foodges_value:fr:4.2
 <en:cream
 en:pasteurized cream
 de:pasteurisierte Sahne
-fr:crème pasteurisé
+fr:crème pasteurisé, crème pasteurisée
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]:
```suggestion
fr:crème pasteurisée
```

"crème pasteurisé" is incorrect

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -1480,7 +1482,7 @@ wikidata:en:Q191907
 wikipedia:en:https://en.wikipedia.org/wiki/Carotenoid
 # ingredient/fr:caroténoïde has 2195 products in 12 languages @2019-04-29
 
-
+<en:paprika
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]: paprika extract is not really paprika, we don't want a parent for it. It's the same for things like orange flavouring, we don't want orange as a parent.

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -25965,7 +25980,7 @@ pt:farinha de centeio
 en:wholemeal rye flour
 de:Roggenvollkornmehl
 es:flor de harina integral de centeno
-fr:farine de seigle complet
+fr:farine de seigle complèt, farine complète de seigle
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]:
```suggestion
fr::farine de seigle complète, farine complète de seigle
```

File: taxonomies/ingredients.txt [hunk 4]
```diff
@@ -34142,12 +34168,14 @@ nl:stukjes appel
 # 45 products in 4 languages @2018-10-04
 
 <en:apple
-fr:compote de pommes
+fr:compote de pommes, compote de pommes
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]:
```suggestion
fr:compote de pommes
```

File: taxonomies/ingredients.txt [hunk 5]
```diff
@@ -47097,46 +47122,40 @@ carbon_footprint_fr_foodges_ingredient:fr:Épices
...
+en:herbs and spices
+fr:épices et plantes aromatique, épice et plante aromatique, épice et plantes aromatiques, épices et plantes aromatiques, épices et plante aromatique, plantes aromatiques et épices, extraits d'épices et d'herbes, herbe et épice, herbes et épices, épices et herbes, épices et herbes aromatiques, épice et aromate, épices et aromates, épices et aromate, épice et aromates, épices et arômes, aromates et épices
 
+<en:herbs and spices
 fr:extraits d'épices et d'aromates
 
 <en:spice
-fr:plantes aromatiques et épices
-
+en:spice extracts
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]: we should remove the <en:spice parent

File: taxonomies/ingredients.txt [hunk 6]
```diff
@@ -53936,16 +53981,14 @@ wikidata:en:Q1142857
 en:shiitake
 de:shiitake, schwarzer Pilz
 es:setas negro
-fr:shiitake, champignon noir, champignons noirs, shiitakes
+fr:shiitake, champignon noir, champignons noirs, shiitakes, , Lentin du chêne, Lentin comestible, Champignon parfumé
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]:
```suggestion
fr:shiitake, champignon noir, champignons noirs, shiitakes, Lentin du chêne, Lentin comestible, Champignon parfumé
```

File: taxonomies/ingredients.txt [hunk 7]
```diff
@@ -64170,6 +64277,9 @@ fr:pain de mie
 it:pane per toast
 # ingredient/fr:pain-de-mie has 104 products in 4 languages @2019-03-07
 
+<en:breadmeal
+fr:pain de mie complèt
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]:
```suggestion
fr:pain de mie complet
```

File: taxonomies/ingredients.txt (hunk 3 above)
Comment (hangy) [reply taxonomies/ingredients.txt]: Why `::`?

Comment (stephanegigandet) [reply taxonomies/ingredients.txt]: my mistake, just one :

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (aleene) [reply taxonomies/ingredients.txt]: I wonder if there are entries with just é.

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (aleene) [reply taxonomies/ingredients.txt]: Just for paprika or all extracts?

File: taxonomies/ingredients.txt (hunk 3 above)
Comment (aleene) [reply taxonomies/ingredients.txt]: Maybe add complèt to be complete.

File: taxonomies/ingredients.txt (hunk 5 above)
Comment (aleene) [reply taxonomies/ingredients.txt]: The idea is to have to parents spice and herb.

**Comment by teolemon** [comment]
@aleene merge conflict

**Comment by teolemon** [comment]
@stephanegigandet paprika question to answer

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (stephanegigandet) [reply taxonomies/ingredients.txt]: @aleene : there may be badly spelled entries, but we should not add them as synonyms, as we will use the taxonomy to do things like spell correction etc.

**Comment by stephanegigandet** [comment]
Going to merge master in and try to make all the tests pass.

**Comment by aleene** [comment]
I'll continue adding ingredients.

**Pull request merged**